rest/service.go

```diff
@@ -321,11 +321,6 @@ func (s *Service) addRoutes() {
 	s.app.AddRoute("/test_results/tasks").Version(1).Get().RouteHandler(makeGetTestResultsByTasks(s.sc))
 	s.app.AddRoute("/test_results/tasks/stats").Version(1).Get().RouteHandler(makeGetTestResultsStatsByTasks(s.sc))
 	s.app.AddRoute("/test_results/tasks/failed_sample").Version(1).Get().RouteHandler(makeGetTestResultsFailedSampleByTasks(s.sc))
-	// TODO (EVG-18798): Remove these test results routes once Spruce and
```

Comment:
Are we certain we're good to remove these? They're [documented](https://github.com/evergreen-ci/cedar/wiki/Rest-V1-Usage#id6) as being part of the API, for instance.

Reply:
They aren't used directly by users (they go through Evergreen rest V2 to fetch test results). Since the app is not longer using them we can just remove them, I will update the documentation as well.